src/runtime/manifest.ts

```diff
@@ -861,7 +861,10 @@ ${e.message}
         if (arg.type.getEntitySchema()) {
           const fields = arg.type.getEntitySchema().fields;
           for (const name of Object.keys(fields)) {
-            fields[name].annotations = Manifest._buildAnnotationRefs(manifest, fields[name].annotations);
+            // Annotations can sometime be already processed in the case of external schemas.
+            if(fields[name].annotations && !(fields[name].annotations[0] instanceof AnnotationRef)) {
```

Comment:
why are you checking `instanceof`, can it be an instance of some other type?

Reply:
it could be AstNode.AnnotationRef, in which case we need to convert to AnnotationRef (from annotation.js) which is what the call to _buildAnnotationRefs does. Yes I also find it confusing to have two AnnotationRef classes.

Reply:
Changed to check for the warning instead. PTAL